src/parallel_som.jl

```diff
@@ -334,13 +351,23 @@ function makeClassFreqs(som, vis, classes)
 end
 
 
+
+# Arguments
+
+
 """
-    find_bmu(cod, sampl)
+    findBmu(cod, sampl)
```

Comment:
What are the input arguments here?

Reply:
I would rename this as `findBMU`

Reply:
yeah I was talking about it with oliver this morning, there was a typo. taking care of it